src/clients/authorization/AuthorizationEvaluate.tsx

```diff
@@ -28,6 +28,16 @@ import { defaultContextAttributes } from "../utils";
 import type ResourceRepresentation from "@keycloak/keycloak-admin-client/lib/defs/resourceRepresentation";
 import { useParams } from "react-router-dom";
 import type ScopeRepresentation from "@keycloak/keycloak-admin-client/lib/defs/scopeRepresentation";
+import type { KeyValueType } from "../../components/attribute-form/attribute-convert";
+
+interface EvaluateFormInputs
+  extends Omit<ResourceEvaluation, "context" | "resources"> {
+  applyToResource: boolean;
+  alias: string;
+  authScopes: string[];
+  context: { [key: string]: string };
+  resources: { [key: string]: string }[];
```

Comment:
You can use a [`Record`](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type) type here.
```suggestion
  context: Record<string, string>;
  resources: Record<string, string>[];
```

Reply:
Good call - thanks @jonkoops